translations/translated/updateqm.pri

```diff
@@ -2,12 +2,20 @@
 
 TS_FILES += $$files(mudlet_*.ts)
 
+# The American translation is plurals only and has to be done separately (by
+# hand) so is not to be include in the built-in Qt TRANSLATIONS variable as the
+# default processing of it will not have the needed extra `-pluralonly`
```

Comment:
Could you write some documentation in wiki maybe, how that seperate manual process would look like for each release?

Reply:
:book: Documentation now in Wiki at: [Translating Mudlet - English (American) translation](https://wiki.mudlet.org/w/Translating_Mudlet#English_.28American.29_translation).